src/DataTypes.ts

```diff
@@ -33,7 +33,7 @@ export interface Force {
 
 export interface ForceFrame {
     time: number;
-    forces: Array<Force>; //usually 1, at most 2
+    forces: Array<Force|null>; //usually 1, at most 2
```

Comment:
I was assuming you would just not include the force in the array if it's missing